src/main/java/org/terasology/launcher/repositories/GithubRepositoryAdapter.java

```diff
@@ -64,6 +66,8 @@ static GameRelease fromGithubRelease(GHRelease ghRelease) {
             } else {
                 engineVersion = new Semver(tagName);
             }
+            //TODO: check whether the launcher can fulfil this requirement
```

Comment:
I guess this might do it right now, although I want to get away from using the launcher's JRE for running the game. But using [`Runtime.Version`](https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/lang/Runtime.Version.html) seems interesting (instead of `Semver`).

Reply:
Gave the `Runtime.Version` a quick test. Some observations:
- To construct these, you'll have to use `Runtime.Version.parse` which requires the versions in a particular format. Unfortunately, it its **not** the format used for the releases 😔 While `Runtime.Version.parse("11.0.21+9")` works, `Runtime.Version.parse("17.0.9+9.1")` does not. Also, for Java 8 one has to translate `8u392-b08` to a compatible format like `Runtime.Version.parse("8.0.392+8")`.
- While you can compare two `Runtime.Version`s, it's not that simple to figure out whether one "satisfies" the other. Sure, you can look a the individual parts manually, but ...

Reply:
Therefore, I don't know what exactly I'd want to test here... Compare `Runtime.version().feature()` against `minJavaVersion.getMajor()` and manually account for backwards compatibility?